### PR TITLE
Convert connector to webhook

### PR DESCRIPTION
**Pull request by evelyn9191**

### CODE REVIEW

File: webhook.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+const createErrorMessages = (id, content) => [{ id, type: 'error', content }];
+
+const trycatch = (fn) => (data) => {
+  try { return fn(data); } catch (e) { return createErrorMessages('all', 'Error parsing annotation'); }
+};
+
+const floatValue = ({ value }) => parseFloat(value);
+
+function vatSumCheck(vatTotal, vatDetails) {
+  const sum = vatDetails.reduce((s, i) => s + floatValue(findByPath(i.children, ['vat_detail_tax'])), 0);
```
Comment (janmarsicek) [inline webhook.js]: was this working for you? I think there is `.content` missing as `findByPath` helper returns the whole datapoint object. When you access the content, `floatValue` will pick the value from it...

File: webhook.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  return Math.abs(sum - floatValue(vatTotal)) <= tolerance;
```
Comment (janmarsicek) [inline webhook.js:39]: should be `vatTotal.content` probably...

File: webhook.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  return (found.children && parts.length) ? findByPath(found.children, parts) : found;
+};
+
+const createErrorMessages = (id, content) => [{ id, type: 'error', content }];
+
+const trycatch = (fn) => (data) => {
+  try { return fn(data); } catch (e) { return createErrorMessages('all', 'Error parsing annotation'); }
+};
+
+const floatValue = ({ value }) => parseFloat(value);
```
Comment (janmarsicek) [inline webhook.js]: OK! ideally, `floatValue` function should look like this: `({ content: { value } }) => parseFloat(value)` and you can ignore my previous comments 😄

**Review by janmarsicek — CHANGES_REQUESTED** [review]
also I would find it very useful to provide a testing schema for the queue, I had hard times debugging it and running to `Error parsing annotation` error a lot since my testing schema uses different schema IDs... but if it works with some default schema we use for registration, it is ok 👌

**Comment by evelyn9191** [comment]
It works with the default EU demo schema. But I added it to the code, it's better like this. Thanks for the note!

**Review by janmarsicek — APPROVED** [review]

**Pull request merged**